.github/workflows/build-m1-binaries.yml

```diff
@@ -123,12 +133,19 @@ jobs:
           . packaging/pkg_helpers.bash
 
           if [[ $CHANNEL == "test" ]]; then
+            setup_cuda
             setup_base_build_version
+            setup_macos
           else
+            setup_cuda
             setup_build_version
+            setup_macos
           fi
 
           setup_conda_pytorch_constraint
+          setup_conda_cudatoolkit_constraint
+          setup_visual_studio_constraint
+          setup_junit_results_folder
```

Comment:
Please don't (and if this is needed, then why? (No VisualStudio on Mac, no CUDA ever on M1 nor do we need to setup junit_tests for binary-builds, do we?
```suggestion
```

Reply:
Done Cleaned up